src/main/scala/symsim/concrete/ConcreteQTable.scala

```diff
@@ -0,0 +1,66 @@
+package symsim
+package concrete
+
+import cats.syntax.option.*
+import org.typelevel.paiges.Doc
+import symsim.QTable
+
+trait ConcreteQTable[State, ObservableState, Action]
+        extends QTable[State, ObservableState, Action, Double, Randomized]:
+  this: ConcreteExactRL[State, ObservableState, Action] =>
+
+  import agent.instances.*
+
+  type Scheduler[A] = Randomized[A]
```

Comment:
I don't remember whether I suggested this, but I think this renaming complicates the reading unnecessarily.  The super-trait does not require that a type constructor `Scheduler[_]` exists, so I think we could just go with `Randomized` everywhere in this trait. As we are already doing with `Double` instead of `Reward`.